addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1808,4 +1808,32 @@ options.registry.CoverProperties = options.Class.extend({
         this.$target[0].dataset.filterColor = activeFilterColor || '';
     },
 });
+
+/**
+ * Allows snippets to be moved before the preceding element or after the following
+ */
+options.registry.snippet_move = options.Class.extend({
+
+    //--------------------------------------------------------------------------
+    // Options
+    //--------------------------------------------------------------------------
+
+    /**
+     * Moves the snippet around
+     */
+    moveSnippet: function (previewMode, direction, $opt) {
+        if(previewMode) {
+          return;
+        }
+
+        switch(direction) {
+            case "prev":
```

Comment:
One of our crazy convention in JS is to use `'` instead of `"` for technical strings and `"` for strings we translate.